packages/nns/src/ledger.canister.ts

```diff
@@ -123,6 +128,29 @@ export class LedgerCanister {
     return response.Ok;
   };
 
+  /**
+   * Transfer ICP from the caller to the destination `Account`.
+   * Returns the index of the block containing the tx if it was successful.
+   *
+   * @throws {@link TransferError}
+   */
+  public icrc1Transfer = async (
+    request: Icrc1TransferRequest,
+  ): Promise<BlockHeight> => {
+    // The transaction fee method is not supported by Ledger App yet.
+    if (request.fee === undefined) {
```

Comment:
prefer `isNullish` for consistency

Reply:
If you review just the second commit, you'll see this is actually consistent with the existing code.
We can consider doing a big cleanup in the repo in a separate PR.